Testscripts/Linux/perf_fio.sh

```diff
@@ -176,7 +176,20 @@ CreateRAID0()
 
 ConfigNVME()
 {
-	install_package "nvme-cli"
+    if [ $DISTRO_NAME == "centos" ] || [ $DISTRO_NAME == "rhel" ] || [ $DISTRO_NAME == "oracle" ]; then
```

Comment:
what this all new code is missing is a basic check if nvme-cli binary already exist on the system.
install_package is fine, at best it will update the package to latest version, but you're doing here a lot of work - deps, compile without a check if nvme-cli might already be on the system.
Please consider adding that, maybe another check in the if loop on line 180 along with the distro version being 6.x

Reply:
Added a check if nvme is already installed and also added a new function check_package which availability of package on distro
Moved this block to a function in utils since it is common for all the nvme tests